website_event_questions_open_ended/controllers/main.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.website_event_questions.controllers.main import WebsiteEvent
+
+
+class WebsiteEventController(WebsiteEvent):
+
+    def _process_registration_details(self, details):
+        ''' Process data posted from the attendee details form. '''
+        # FIXME the call to super here goes directly to website_event's controller skipping website_event_questions' controller  # noqa
+        registrations = super(
+            WebsiteEvent, self)._process_registration_details(details)
+        for registration in registrations:
+            open_answers = []
+            for key, value in registration.items():
+                if key.startswith('open-answers-'):
+                    open_answers.append([0, 0, {
+                        'question_id': key[13:],
```

Comment:
:open_mouth: no better way to slice this key?
`key.split('open-answers-')[-1]`?